src/Symfony/Component/Security/Http/Attribute/CurrentUser.php

```diff
@@ -20,8 +20,17 @@
 #[\Attribute(\Attribute::TARGET_PARAMETER)]
 class CurrentUser extends ValueResolver
 {
-    public function __construct(bool $disabled = false, string $resolver = UserValueResolver::class)
-    {
+    public function __construct(
+        /*
+         * Whether to disable thi value resolver.
```

Comment:
What does it mean to disable a value resolver?

```suggestion
         * Whether to disable this value resolver.
```